vaultclient/src/test/java/com/scality/vaultclient/services/AccountServicesClientTest.java

```diff
@@ -0,0 +1,268 @@
+                    .name(null)
+                    .build();
+        }, "Expected NullPointerException");
+    }
+
+    @Test
+    public void createAccountErrorExistingAccount() throws Exception {
+
+        final String ENTITY_EXISTS_ERR = "The request was rejected because it attempted to create a resource that already exists.";
+
+        when(amazonHttpClient.execute(any(Request.class), any(), any(), any(),any()))
+                .thenAnswer(new Answer<Response>() {
+                    @Override
+                    public Response<CreateAccountResponseDTO> answer(InvocationOnMock invocation) {
+                        VaultClientException e = new VaultClientException("EntityAlreadyExists");
+                        e.setErrorCode("EntityAlreadyExists");
+                        e.setErrorMessage(ENTITY_EXISTS_ERR);
+                        e.setStatusCode(409);
+                        e.setServiceName("Vault");
+                        throw e;
+                    }
+                });
+
+        VaultClientException e = assertThrows(VaultClientException.class, () -> {
+            accountServicesClient.createAccount(createAccountRequestDTO);
+        }, "Expected VaultClientException");
+        assertEquals(409, e.getStatusCode(), "Expected http status code: 409");
+        assertEquals("EntityAlreadyExists", e.getErrorCode(), "Expected error code: EntityAlreadyExists");
+        assertEquals(ENTITY_EXISTS_ERR, e.getErrorMessage(), "Invalid error message");
+
+        //reinit the amazonHttpClient
+        init();
+    }
+
+    @Test
+    public void testCreateAccountError400() throws Exception {
+
+        when(amazonHttpClient.execute(any(Request.class), any(), any(), any(),any()))
+                .thenAnswer(new Answer<Response>() {
+                    @Override
+                    public Response<CreateAccountResponseDTO> answer(InvocationOnMock invocation) {
+                        Request<CreateAccountRequestDTO> ogReq = invocation.getArgument(0);
+                        HttpRequestBase httpRequestBase = new HttpRequestBase() {
+                            @Override
+                            public String getMethod() {
+                                return "Not Implemented";
+                            }
+                        };
+                        HttpResponse httpResponse = new HttpResponse(ogReq, httpRequestBase);
+                        httpResponse.setStatusCode(400);
+                        httpResponse.setStatusText("Bad Request");
+                        return new Response<>(new CreateAccountResponseDTO(),httpResponse);
+                    }
+                });
+
+        HttpResponse response = accountServicesClient.createAccount(createAccountRequestDTO).getHttpResponse();
+
+        assertEquals(400, response.getStatusCode(), "Expected http status code: 409");
+
+        //reinit the amazonHttpClient
+        init();
+    }
+
+    @Disabled
```

Comment:
This is a test case to connect to real vault instead of mock application. I disabled and committed it so it can be useful during the future development.